tests/phpunit/tests/blocks/getBlockTemplates.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * @group blocks
+ *
+ * @covers ::get_block_templates
+ */
+class Tests_Blocks_GetBlockTemplates extends WP_UnitTestCase {
+
+	const TEST_THEME = 'block-theme';
+
+	/**
+	 * @var WP_Post
+	 */
+	private static $template;
+
+	/**
+	 * @var WP_Post
+	 */
+	private static $template_part;
+
+	public static function wpSetUpBeforeClass() {
+
+		// This template has to have the same ID ("block-theme/index") as the template that is shipped with the
+		// "block-theme" theme. This is needed for testing purposes.
+		static::$template = self::factory()->post->create_and_get(
+			array(
+				'post_type' => 'wp_template',
+				'post_name' => 'index',
+				'tax_input' => array(
+					'wp_theme' => array(
+						static::TEST_THEME,
+					),
+				),
+			)
+		);
+		wp_set_post_terms( static::$template->ID, static::TEST_THEME, 'wp_theme' );
+
+		// This template part has to have the same ID ("block-theme/small-header") as the template part that is shipped with the
+		// "block-theme" theme. This is needed for testing purposes.
```

Comment:
```
/*
 * This template part has to have the same ID ("block-theme/small-header") as the template part that is shipped with the
 * "block-theme" theme. This is needed for testing purposes.
 */
```

Reply:
Fixed in b15a8a4449a98a1cf4dcffa5869c00c3b2de3276